packages/game-client/src/App.jsx

```diff
@@ -168,6 +175,16 @@ function App() {
 		directions.current[snakeId] = direction;
 	};
 
+	//gets all the hashes of snakes and returns it as an object :)
+
+	const getSnakeHash = (state) => {
```

Comment:
Can name this function something like getSnakeCells(), since it gives you the cells which are occupied by snakes.